src/main/java/com/coherentsolutions/training/auto/web/pashkovskaya/pages/CartPage.java

```diff
@@ -0,0 +1,20 @@
+package com.coherentsolutions.training.auto.web.pashkovskaya.pages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class CartPage extends BasePage{
+    @FindBy(xpath = "//tr[@class='totals sub'] //span[@class='price']")
+    public WebElement totalPrice;
+    public CartPage(WebDriver driver) {
+        super(driver);
+    }
+
+    public float getTotalPriceFromCart() {
```

Comment:
try to test Assert.assertEquals(1.0f, 0.5f + 0.5f)
never use floating point datatypes with money, java has BigDecimal

Reply:
java has BigDecimal type for money